app/services/albums.js

```diff
@@ -0,0 +1,13 @@
+const rp = require('request-promise');
+
+module.exports.listAlbums = () => {
+  rp('https://jsonplaceholder.typicode.com/albums')
+    .then(response => console.log(response))
```

Comment:
`.then` is not necessary

Reply:
This will be applied in the next pull request.